cypress/e2e/login.cy.js

```diff
@@ -0,0 +1,23 @@
+const USER_EMAIL = "ronswanson@noroff.no";
```

Comment:
Since this is real data, you should really put it in `.env`.

The easiest way to do this is by importing `dotenv/config` into `cypress.config.js` and embedding these variables in the `env` block of cypress setup config.

Reply:
Changes has been made.  I enjoyed learning this, and see the value in executing these changes.